cocos/2d/CCActionManager.cpp

```diff
@@ -398,6 +398,32 @@ ssize_t ActionManager::getNumberOfRunningActionsInTarget(const Node *target) con
     return 0;
 }
 
+// FIXME: Passing "const O *" instead of "const O&" because HASH_FIND_IT requires the address of a pointer
+// and, it is not possible to get the address of a reference
+size_t ActionManager::getNumberOfRunningActionsInTargetByTag(const Node *target,
```

Comment:
I have the same feel, but I just  followed the pattern in the code!
The other methods are "overloaded" by adding the "byTag" suffix, so I did the same.

For the other side, I think that is easier to make a mental model with this name since
we are used to the other methods that uses the "byTag" suffix.